src/Aspire.Hosting.AWS/Utils/ProjectUtilities.cs

```diff
@@ -0,0 +1,123 @@
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+namespace Aspire.Hosting.AWS.Utils;
+
+internal static class ProjectUtilities
+{
+    /// <summary>
+    /// Initializes the project's launch settings if necessary, and
+    /// ensures they are referencing the Amazon.Lambda.TestTool's location.
+    /// </summary>
+    public static void UpdateLaunchSettingsWithLambdaTester(
+        string resourceName, 
+        string functionHandler, 
+        string assemblyName, 
+        string projectPath, 
+        string runtimeSupportAssemblyPath, 
+        string targetFramework)
+    {
+        // Retrieve the current launch settings JSON from wherever it's stored.
+        string launchSettingsJson = GetLaunchSettings(projectPath);
+
+        // Parse the JSON into a mutable JsonNode (root is expected to be an object)
+        JsonNode? rootNode = JsonNode.Parse(launchSettingsJson);
```

Comment:
Can you catch JSON parse exceptions in case the launchsettings json file is malformed and if so log an error and return.